src/extension.ts

```diff
@@ -1341,34 +1341,64 @@ export class ExtensionManager implements vscode.Disposable {
         return this.runCMakeCommandForProject(cmakeProject => cmakeProject.quickStart(folder));
     }
 
-    launchTargetPath(folder?: vscode.WorkspaceFolder | string) {
+    launchTargetPath({ folder, targetName }: { folder?: vscode.WorkspaceFolder | string; targetName?: string }) {
         telemetry.logEvent("substitution", { command: "launchTargetPath" });
-        return this.queryCMakeProject(cmakeProject => cmakeProject.launchTargetPath(), folder);
+        return this.queryCMakeProject(async cmakeProject => {
+            if (targetName !== null) {
```

Comment:
These should probably be checking for `undefined` rather than `null`. If you want to be safe, feel free to check for both, with `undefined` coming first.